vehicles/vehiclestate.h

```diff
@@ -74,6 +74,8 @@ class VehicleState : public ObjectState
     void setAutopilotRadius(double radius);
     double getAutopilotRadius();
 
+    QPointF transformENUPointToVehicleFrame(const QPointF &point, PosType vehiclePosType);
```

Comment:
This might be better located in core/coordinateTransforms.h and get the vehicleState as additional parameter? Unsure, you decide. 🙂